Sources/Sentry/include/SentryFramesTracker.h

```diff
@@ -17,6 +20,17 @@ SENTRY_NO_INIT
 @property (nonatomic, assign, readonly) SentryScreenFrames *currentFrames;
 @property (nonatomic, assign, readonly) BOOL isRunning;
 
+#    if SENTRY_TARGET_PROFILING_SUPPORTED
+/**
+ * The tracer that is currently using this frame tracker. Provided so that the frame tracker can
+ * query for whether a profiler is currently running.
+ */
+@property (nullable, nonatomic, weak) SentryTracer *currentTracer;
```

Comment:
`h`: It can be that there are multiple tracers running at the same time. We currently add slow and frozen frames for every transaction. I think we should change this to a list of activeTracers.

Reply:
We currently only profile the first transaction created, so i've moved the statement that sets `currentTracer` into the same block as the rest of the logic that starts the profiler.